### ISSUE TITLE
Rework README

### ISSUE THREAD

**Issue by solardiz** [openwall/passwdqc#20]
Our current `README` dates back to when this was only a PAM module. Now we also have a library and tools. We should replace the contents of `README` to describe the whole project as it currently is, probably moving most of what's currently in `README` to a new file - e.g., call it `OPTIONS`?

**Comment by ldv-alt** [comment]
Yes, this makes sense, there should be a file describing the whole project, but I'm not volunteering to do this job. :)
Given that options are also described in `passwdqc.conf(5)` manual page, maybe the file describing `pam_passwdqc` should rather focus on other aspects, e.g. pam stack integration.

**Comment by solardiz** [comment]
I was thinking that we wouldn't have a separate text "file describing `pam_passwdqc`" - rather, the options list moved from `README` to e.g. `OPTIONS` would describe the configuration options most of which are not specific to the PAM module alone, but are also supported across other components. Yes, it would be duplicate with `passwdqc.conf(5)` - just different format.

Currently, PAM stack integration would fit (albeit not perfectly) into PAM-using systems' sections in `PLATFORMS`.
